python/QC3.py

```diff
@@ -0,0 +1,31 @@
+from subprocess import CalledProcessError
+from wrappers import runCommand
+from options import parser
+
+args = parser.parse_args()
+cmd = ["python","$GEM_BASE/python/Produce_Config_File.py"]
```

Comment:
From the same file that you took `runCommand()` also take `envCheck()` and then do a check that the `$GEM_BASE` variable has been declared.

e.g.:

```
envCheck("GEM_BASE")
```

Should throw an exception if it is not declared.